database/postgresql.js

```diff
@@ -0,0 +1,31 @@
+require('dotenv').config();
+// const fs = require('fs');
+const {Pool, Client} = require('pg');
+// const copyFrom = require('pg-copy-streams').from;
+
+const connectionString = process.env.POSTGRES_CONNECTION_STRING;
+console.log(connectionString);
+
+const pool = new Pool({
+  connectionString: connectionString,
+});
+
+pool.query('SELECT NOW()', (err, res) => {
+  console.log(err, res);
+  pool.end();
+});
+
+const client = new Client ({
+  connectionString: connectionString,
+});
+
+client.connect();
+
+client.query('SELECT NOW()', (err, res) => {
+  console.log(err, res);
+  client.end();
+});
```

Comment:
lgtm. i personally don't really know the difference of pool, and client at the moment, although i think i have read that using pools can increase query time, not sure about that, but something like that they don't require closing connections, ... not really sure though. anyway i am just using pool currently. let me know if there is a need to using 'client'.